admin-dev/themes/new-theme/template/components/layout/employee_dropdown.tpl

```diff
@@ -39,12 +39,21 @@
     </div>
 
     <p class="divider"></p>
-    <a class="dropdown-item" href="{l s='https://www.prestashop.com/en/resources/documentations?utm_source=back-office&utm_medium=profile&utm_campaign=resources-en&utm_content=download17' d='Admin.Navigation.Header'}" target="_blank" rel="noreferrer"><i class="material-icons">book</i> {l s='Resources' d='Admin.Navigation.Header'}</a>
-    <a class="dropdown-item" href="{l s='https://www.prestashop.com/en/training?utm_source=back-office&utm_medium=profile&utm_campaign=training-en&utm_content=download17' d='Admin.Navigation.Header'}" target="_blank" rel="noreferrer"><i class="material-icons">school</i> {l s='Training' d='Admin.Navigation.Header'}</a>
-    <a class="dropdown-item" href="{l s='https://www.prestashop.com/en/experts?utm_source=back-office&utm_medium=profile&utm_campaign=expert-en&utm_content=download17' d='Admin.Navigation.Header'}" target="_blank" rel="noreferrer"><i class="material-icons">person_pin_circle</i> {l s='Find an Expert' d='Admin.Navigation.Header'}</a>
-    <a class="dropdown-item" href="{l s='https://addons.prestashop.com?utm_source=back-office&utm_medium=profile&utm_campaign=addons-en&utm_content=download17' d='Admin.Navigation.Header'}" target="_blank" rel="noreferrer"><i class="material-icons">extension</i> {l s='PrestaShop Marketplace' d='Admin.Navigation.Header'}</a>
-    <a class="dropdown-item" href="{l s='https://www.prestashop.com/en/contact?utm_source=back-office&utm_medium=profile&utm_campaign=help-center-en&utm_content=download17' d='Admin.Navigation.Header'}" target="_blank" rel="noreferrer"><i class="material-icons">help</i> {l s='Help Center' d='Admin.Navigation.Header'}</a>
-    <p class="divider"></p>
+
+    {foreach from=$displayBackOfficeEmployeeMenu item=$menuItemModule name=displayBackOfficeEmployeeMenu}
+      {foreach from=$menuItemModule item=$menuItem}
+        <a
+          class="dropdown-item"
+          href="{$menuItem.link}"
+          target="_blank"
+          rel="noreferrer"
+        >{if isset($menuItem.icon)}<i class="material-icons">{$menuItem.icon}</i> {/if}{$menuItem.title}</a>
```

Comment:
Should be in a dedicated template to avoid duplicate.

Reply:
Can't. As it in two dedicated themes.